docs/community/com-query/grafana-connector.md

```diff
@@ -203,6 +180,36 @@ which is not a valid query. Instead, you'll want to use it as a child of another
 - User input:  `Intersect(ConstRow(columns=$var3), Row(region="NW"))`
 - Interpolated query: `Intersect(ConstRow(columns=[5,10,15]), Row(region="NW"))`
 
+### Variables in SQL
```

Comment:
Are these examples, syntax, important information?
Is there a reason they're here?
What's the context?

Reply:
@rachithrr you can skip this